### PR TITLE
Slovenian language translation complete

### PR DESCRIPTION
**Pull request by matjazh2**
I finished the rest of the translation and re-checked everything all over again... Took me a while but I found a few mistakes I did on the way without knowing (mostly just minimal pronunciation mistakes and missed caps). Now should be ready to go for publishing.
You can also put me in "Thanks to": Matjaž Hribljan (sl)

### CODE REVIEW

File: src/lang/sl.json [hunk 1]
```diff
@@ -1016,126 +1016,126 @@
...
+  "Show model power curve": "Prikažite krivuljo moči modela",
+  "Show notes and comments": "Prikažite opombe in komentarje",
+  "Show paired workout": "Prikažite združene treninge",
+  "Show past planned workouts": "Prikažite pretekle planirane treninge",
+  "Show power / HR": "Prikažite moč / HR",
+  "Show ride eFTP": "Prikažite eFTP vožnje",
+  "Show sub-maximal efforts": "Prikažite podmaksimalne napore",
+  "Show time in zone as %": "Prikažite čas v pasu kot %",
+  "Show time in zones": "Prikažite čas v pasovih",
   "Show week": "Predstavitveni teden",
```
Comment (matjazh2) [inline src/lang/sl.json:1084]: "Show week": "Prikažite teden", -> correct one

**Comment by davidtinker** [comment]
Thanks a stack! Will do!

**Comment by davidtinker** [comment]
I noticed that at least one sl word is "too long" compared to the English original: Obremenitev

Screen Shot 2022-03-14 at 19 39 08

**Comment by matjazh2** [comment]
Hmm... Maybe i can shorten it somehow... Let me think

**Pull request merged**